distributed/tests/test_scheduler.py

```diff
@@ -2057,3 +2057,14 @@ def reducer(x, y):
     s.validate_state()
     assert set(s.workers) == {a.address}
     assert s.aliases == {a.name: a.address}
+
+
+@gen_cluster(client=True, config={"distributed.scheduler.unknown-task-duration": "1h"})
+async def test_unknown_task_duration_config(client, s, a, b):
+    future = client.submit(inc, 1)
+    while not s.tasks:
+        await asyncio.sleep(0.001)
+    assert sum(s.get_task_duration(ts) for ts in s.tasks.values()) == 3600
+    assert len(s.unknown_durations) == 1
```

Comment:
What if `inc` finishes before we get here?

Reply:
You might want to consider `slowinc`

Reply:
Good point. It should be easy to switch to `slowinc` here.